internal/cmd/root.go

```diff
@@ -47,3 +68,10 @@ operators from the installed catalogs.`,
 
 	return cmd
 }
+
+func setContext(cmd *cobra.Command, ctx context.Context) { //nolint:golint
```

Comment:
Looks funny, but this is the only way to set the unexported field I guess. 

One thing worth noting is mentioned in https://github.com/spf13/cobra/pull/1118#discussion_r562959128 where this root command context would be propagated down to all children. This is what we want via this PR obviously, and if the user wanted to change it on a per-invocation basis they can set the `--timeout` flag on the CLI to whatever. This change would also encourage to build smaller command surfaces that can be used independently of others. 

so a `kubectl operator <command> --timeout 5m | kubectl operator <command> --timeout 10m` could be appropriate in certain cases